### PR TITLE
refine square_error_cost layer

### PR DESCRIPTION
**Pull request by QiJune**

### CODE REVIEW

File: python/paddle/v2/framework/tests/op_test.py [hunk 1]
```diff
@@ -295,15 +295,16 @@ def find_actual(target_name, fetch_list):
                             + ") has different lod at " + str(place))
             else:
                 idx = find_actual(out_name, fetch_list)
-                actual_t = outs[idx]
+                actual = outs[idx]
+                actual_t = np.array(actual)
```
Comment (tonyyang-svail) [inline python/paddle/v2/framework/tests/op_test.py]: what is the purpose of these changes?

Comment (QiJune) [reply python/paddle/v2/framework/tests/op_test.py]: Because type of outs[idx] is LoDTensor, not numpy array. We should convert it to numpy array before comparing.
